src/content/docs/agents/php-agent/php-agent-api/index.mdx

```diff
@@ -12,7 +12,7 @@ Among other things, you can use the API to [customize your app name](/docs/agent
 Enclose these API calls in a conditional check for the PHP agent so that your code will work even if the agent is not present:
 
 ```php
-if (extension_loaded('newrelic')) {
+  if (extension_loaded('newrelic')) {
     newrelic_set_appname($name);
   }
```

Comment:
```suggestion
if (extension_loaded('newrelic')) {
  newrelic_set_appname($name);
}
```

Sorry, should have specified that I meant the other way. I think this may show up as indented on that page, which we don't want.